confd/service/service_test.go

```diff
@@ -73,7 +81,8 @@ func TestCreateService(t *testing.T) {
 		raw["name"] = "heartOfGold"
 		raw["service"] = "8.8.8.8"
 
-		service := createService(raw)
+		service, err := createService(raw)
+		assert.NoError(t, err)
 		assert.Equal(t, "heartOfGold", service.Location)
```

Comment:
Always use `require.Equal` when not fulfilling leads to a segmentation fault (null pointer)